src/main/java/teammates/ui/webapi/Action.java

```diff
@@ -111,6 +113,27 @@ public void checkAccessControl() throws UnauthorizedAccessException {
         checkSpecificAccessControl();
     }
 
+    /**
+     * Gets the user information of the current user.
+     */
+    public Map<String, String> getUserInfo() {
```

Comment:
Be careful of naming the method like this. There is a `userInfo` field in this class.

Reply:
Ok. It is updated.